readme.md

```diff
@@ -27,6 +27,13 @@ pupa('The mobile number of {name} is {phone.mobile}', {
 
 pupa('I like {0} and {1}', ['🦄', '🐮']);
 //=> 'I like 🦄 and 🐮'
+
+// Double braces encodes the HTML entities to avoid code injection.
+pupa('{{0}}{{1}}', ['<br>yo</br>', '<i>lol</i>']);
+//=> '&lt;b&gt;yo&lt;/b&gt;&lt;b&gt;lol&lt;/b&gt;
```

Comment:
I would prefer reusing the example on line 28, just wrap the emojis in HTML tags.